features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Applications/Details/index.js

```diff
@@ -18,48 +18,34 @@
 
 import React, {Component} from 'react'
 import PropTypes from 'prop-types';
+import {Route, Switch, Redirect, Link} from 'react-router-dom'
+import { withStyles } from '@material-ui/core/styles';
 import ProductionKey from './ProductionKey'
 import SandboxKey from './SandboxKey'
 import Subscriptions from './Subscriptions'
-import {Route, Switch, Redirect, Link} from 'react-router-dom'
 import API from '../../../data/api'
 import {PageNotFound} from '../../Base/Errors/index'
 import Loading from '../../Base/Loading/Loading'
 import ResourceNotFound from "../../Base/Errors/ResourceNotFound";
-import { withStyles } from '@material-ui/core/styles';
 import CustomIcon from '../../Shared/CustomIcon';
 import InfoBar from './InfoBar';
-
-
+import LeftMenuItem from '../../Shared/LeftMenuItem';
```

Comment:
Shall we define an alias 'AppComponents' as in publisher and use it for imports?